paket.lock

```diff
@@ -297,6 +303,9 @@ NUGET
       Grpc (>= 2.44 < 3.0) - restriction: || (&& (== net6.0) (>= net462)) (&& (== net6.0) (< netstandard2.1)) (&& (== net7.0) (>= net462)) (&& (== net7.0) (< netstandard2.1)) (== netstandard2.0)
       Grpc.Net.Client (>= 2.43 < 3.0) - restriction: || (== net6.0) (== net7.0) (&& (== netstandard2.0) (>= netstandard2.1))
       OpenTelemetry (>= 1.3.2)
+    Ply (0.3.1) - restriction: || (&& (== net6.0) (< netstandard2.1)) (&& (== net7.0) (< netstandard2.1)) (== netstandard2.0)
+      FSharp.Core (>= 4.6.2)
+      System.Threading.Tasks.Extensions (>= 4.5.4)
```

Comment:
FsToolkit uses Ply in netstandard2.0 only, this is mostly superfluous.

Reply:
What do we have left in NS2.0 - should we wholesale go over to net6.0/net7.0?

Reply:
I don't think any of our projects target NS2.0 - we probably just haven't removed it in `paket.deps`

Reply:
handled in https://github.com/fsharp/FsAutoComplete/pull/1088/commits/88731dc87098286c0f75e33a972952dc29962885